hw/dv/sv/cip_lib/doc/index.md

```diff
@@ -412,6 +412,34 @@ class ip_env_cfg extends cip_base_env_cfg #(.RAL_T(ip_reg_block));
     tl_intg_alert_fields[ral.fault_status.intg_err] = 1;
 ```
 
+### Security Verification for memory integrity
+The countermeasure of memory integrity stores the data integrity in the memory rather than generating the integrity on-the-fly during a read.
+The test [passthru_mem_intg_tests](https://github.com/lowRISC/opentitan/blob/master/hw/dv/tools/dvsim/tests/passthru_mem_intg_tests.hjson) can fully verified this countermeasure.
```

Comment:
Grammar nit: At the moment, this says "The test passthru_mem_intg_tests ...", which looks odd because we're duplicating "test" and also the singular and plural don't match. Maybe just "The passthru_mem_intg_tests ..." ?

Also, this should be "can fully verif**y** this countermeasure."